src/main/java/io/github/lambdatest/gradle/AppUploader.java

```diff
@@ -6,10 +6,11 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * Handles the asynchronous upload of application files to the LambdaTest platform.
- * This class manages the upload process and returns the application ID for test execution.
+ * Handles the asynchronous upload of application files to the LambdaTest platform. This class
+ * manages the upload process and returns the application ID for test execution.
```

Comment:
Again, I think the formatter is broken. Let me know if I should undo the formatter actions (CI will complain, though).

Reply:
No, this is also fine.